docs/howto/openstack/glance/filter.md

```diff
@@ -15,10 +15,11 @@ You can use either the `openstack` command-line interface (CLI), or the {{gui}},
     ## On the Images tab
 
     You can also see the full list of images in the *Images → Public Images* menu item.
-    However, you can only filter by the image name on this page.
-    No other filtering or sorting is available at the moment.
+    On this page you can filter images based on their properties, like name, uuid or even tags.
```

Comment:
This is confusing because it seems to mix the generic meaning of "property" with the meaning that is specific to Glance images. Please rephrase to make that distinction clearer.